pkg/clusterversion/cockroach_versions.go

```diff
@@ -777,6 +779,10 @@ var versionsSingleton = func() keyedVersions {
 // simply need to check is that the cluster has upgraded to 23.1.
 var V23_1 = versionsSingleton[len(versionsSingleton)-1].Key
 
+const (
+	BinaryMinSupportedVersionKey = V22_1
```

Comment:
https://github.com/cockroachdb/cockroach/blob/b258d20a11f66c498309d87f53a89732dd267c7c/pkg/clusterversion/clusterversion.go#L241-L244 is the way to do it I do believe

Reply:
fwiw I did not end using `BinaryMinSupportedVersionKey` because it's different from the tenant creation min version but I decided to keep this, assuming we will unify them? I can remove it. We need the key specifically rather than the version because we map the key to the initial values we want from https://github.com/cockroachdb/cockroach/blob/af0beec136262540f2e777c1b52a93ad21ac7481/pkg/sql/catalog/bootstrap/previous_releases.go#L26

Reply:
ah, I see. I guess what we could do is just write a test that asserts that the key here maps to the min binary version or something like that.

Reply:
and in a similar test (or in the same one) ensure that when [BinaryVersionKey](https://github.com/cockroachdb/cockroach/blob/af0beec136262540f2e777c1b52a93ad21ac7481/pkg/clusterversion/cockroach_versions.go#L797) is passed to `ByKey` it returns the same version as [binaryVersion](https://github.com/cockroachdb/cockroach/blob/af0beec136262540f2e777c1b52a93ad21ac7481/pkg/clusterversion/cockroach_versions.go#L801)?